content/Software Support/Installation/Error-bad-CPU-type-in-executable.md

```diff
@@ -0,0 +1,47 @@
+---
+title: 'Error "bad CPU type in executable"'
+---
+
+Under certain conditions, startup or other operations of the Arduino development software might fail with a "`bad CPU type in executable`" error.
+
+This problem can be solved by one of the methods described below:
+
+## If your Mac computer has an Apple Silicon processor
+
+Some of the tools used by the Arduino development software are built for use on "Intel" processors. These can still be used on the [Apple Silicon](https://support.apple.com/en-us/HT211814) Mac computers thanks to a program named **Rosetta** that Apple provides.
+
+**Rosetta** is not pre-installed with macOS. If you have not already installed **Rosetta**, the tools built for use on "Intel" processors will fail to run with a "`bad CPU type in executable`" error.
+
+The solution is to install **Rosetta** by one of the following methods:
+
+### Installation Dialog
+
+When you start an application that requires it, macOS will show [a dialog offering installation of **Rosetta**](https://support.apple.com/en-us/HT211861). If you are presented with this dialog when starting the Arduino development software, click the ![Install button](img/symbol_install-rosetta.png) button and complete the installation.
+
+### Manual Installation
+
+If the tool built for use on "Intel" processors is run by the Arduino development software as a subprocess, macOS does not show a dialog offering installation of **Rosetta**. In this case you must manually install **Rosetta**:
+
+1. Open **Terminal** (instructions [here](https://support.apple.com/en-ca/guide/terminal/apd5265185d-f365-44cb-8b09-71a064a42125/mac#trml663c6997)).
+1. Type the following command in the **Terminal** window:
+
+   ```text
+   softwareupdate --install-rosetta
+   ```
+
+1. Press the <kbd>**Enter**</kbd> key.
+1. Accept the license agreement with Apple.
+1. Wait for the installation to finish.
+
+## If computer has other processor or Rosetta already installed
```

Comment:
I originally had a more verbose heading text "If your computer has a different processor or Rosetta is already installed" just as you suggest here, but this violated the `HC001` linter rule:

https://github.com/arduino/help-center-content/actions/runs/4113082730/jobs/7098776881#step:3:27

```text
34: HC001: Heading length above limit [## If your computer has a different processor or Rosetta is already installed]
```

So I was forced to make drastic cuts to the text. Since your suggestions are longer than the allowed 60 character length, I can't accept them, but I welcome any suggestions for how the heading text can be improved while staying within limit imposed by the linter.

Reply:
Sounds good.

Resolved by https://github.com/arduino/help-center-content/compare/21fdad63e857371ddfa0eeb80f72cb0b0f4e4bd7..7d08d12db78771b98543f90b095d11f402f808d9